src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php

```diff
@@ -221,7 +225,11 @@ public function update(): EndpointResourceResult
     {
         $modules = self::CONFIGURABLE_MODULES;
         foreach ($modules as $key => $module) {
-            $modules[$key] = $this->getRequestParams()->getBoolean(RequestParams::PARAM_TYPE_BODY, $key, true);
+            if ($key === self::PARAMETER_CLAIM) {
+                $modules[$key] = $this->getRequestParams()->getBoolean(RequestParams::PARAM_TYPE_BODY, $key, false);
+            } else {
+                $modules[$key] = $this->getRequestParams()->getBoolean(RequestParams::PARAM_TYPE_BODY, $key, true);
+            }
```

Comment:
Seems we can revert this as previous

```suggestion
            $modules[$key] = $this->getRequestParams()->getBoolean(RequestParams::PARAM_TYPE_BODY, $key, self::CONFIGURABLE_MODULES[$key] );
```
why can't we do like this.